Android/Sanku_Yogesh/Advanced/newfeedapp/app/src/main/java/android/example/newsfeedapp/NewsDetailActivity.java

```diff
@@ -0,0 +1,129 @@
+package android.example.newsfeedapp;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+
+import android.content.Intent;
+import android.media.Image;
+import android.os.Bundle;
+import android.view.View;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.load.resource.drawable.DrawableTransitionOptions;
+import com.bumptech.glide.request.RequestOptions;
+import com.google.android.material.appbar.AppBarLayout;
+import com.google.android.material.appbar.CollapsingToolbarLayout;
+
+public class NewsDetailActivity extends AppCompatActivity implements  AppBarLayout.OnOffsetChangedListener{
+
+    private ImageView imageView;
+    private TextView appbar_title, appbar_subtitle, date, time, title;
+    private boolean isHideTolbarView = false;
+    private FrameLayout date_behavior;
+    private LinearLayout titleAppbar;
+    private AppBarLayout appBarLayout;
+    private Toolbar toolbar;
+    private String mUrl, mImg, mTitle, mDate, mSource, mAuthor;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_news_detail);
+        toolbar = findViewById(R.id.toolbar);
+        //setSupportActionBar(toolbar);
+        getSupportActionBar().setTitle("");
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
```

Comment:
Method invocation 'setDisplayHomeAsUpEnabled' and setTitle() may produce 'NullPointerException'. Please handle.